tests/sentry/event_manager/test_severity.py

```diff
@@ -75,6 +77,21 @@ def test_error_event_simple(self, mock_urlopen: MagicMock) -> None:
         assert reason == "ml"
         assert cache.get(SEER_ERROR_COUNT_KEY) == 0
 
+        with override_options({"seer.api.use-shared-secret": 1.0}), override_settings(
+            SEER_API_SHARED_SECRET="some-secret"
+        ):
+            _get_severity_score(event)
+            mock_urlopen.assert_called_with(
+                "POST",
+                "/v0/issues/severity-score",
+                body=orjson.dumps(payload),
+                headers={
+                    "content-type": "application/json;charset=utf-8",
+                    "Authorization": "Rpcsignature rpc0:8d982376e4e49ffe845ed39853f6f2cb9bf38564d2a8a325dcd88abba8c58564",
+                },
```

Comment:
QQ: what is that signature format?

Reply:
An arbitrary format that RPC has settled on for now: https://github.com/search?q=repo%3Agetsentry%2Fsentry%20Rpcsignature&type=code

I suspect in the future the mechanism and protocol of RPC will change to a standard and not an inhouse implementation, which will simplify things, but for the time being this will work.

🙏 thanks for the review